packages/backend/src/config/config.staging.ts

```diff
@@ -15,5 +16,27 @@ export function getStagingConfig(): Config {
       application_name: name,
     },
     tvlReportSync: true,
+    activityV2: {
+      starkexApiKey: getEnv('STARKEX_API_KEY'),
+      starkexApiDelayHours: 12,
+      starkexCallsPerMinute: 200,
+      projects: {
```

Comment:
Don't we want to run all projects on staging?

Reply:
We want - this parameter is used to override configuration from `config` package only.